projects/mac_ai/testing/llama_cpp.py

```
@@ -128,7 +128,7 @@ def rebuild_image(start=True):
 def start_podman():
     base_work_dir = remote_access.prepare()
 
-    prepare_test(base_work_dir, utils.parse_platform("macos/llama_cpp/upstream"))
+    prepare_test(base_work_dir, utils.parse_platform("macos/llama_cpp/upstream_bin"))
```

Ruff (0.8.2)

Undefined name `prepare_test`

(F821)